### PR TITLE
JBPM-9286 UI error when parent process instance details have been rem…

### PR DESCRIPTION
**Pull request by bxf12315**
…oved

**Thank you for submitting this pull request**

**JIRA**: _(JBPM-9286)_ 

[link](https://issues.redhat.com/browse/JBPM-9286)

**referenced Pull Requests**: _(please edit the URLs of referenced pullrequests if they exist)_

https://github.com/kiegroup/droolsjbpm-integration/pull/2193

**How to retest or run**:

* a pull request please add comment: regex **[.\*[j|J]enkins,?.\*(retest|test) this.\*]**

* a full downstream build please add comment: regex **[.*\[j|J]enkins,?.\*(execute|run|trigger|start|do) fdb.\*]**

* a compile downstream build please  add comment: regex **[.\*[j|J]enkins,?.\*(execute|run|trigger|start|do) cdb.\*]**

* a full production downstream please add comment: regex **[.\*[j|J]enkins,?.\*(execute|run|trigger|start|do) product fdb.\*]**

* an upstream build please add comment: regex **[.\*[j|J]enkins,?.\*(execute|run|trigger|start|do) upstream.\*]**

i.e for running a full downstream build =  **Jenkins do fdb**

### CODE REVIEW

File: jbpm-wb-process-runtime/jbpm-wb-process-runtime-client/src/test/java/org/jbpm/workbench/pr/client/editors/instance/details/ProcessInstanceDetailsTabPresenterTest.java [hunk 1]
```diff
@@ -111,6 +115,15 @@ public void setProcessInstanceDetailsTest() {
                           nodeInstanceSummary.getType());
     }
 
+    @Test
+    public void setProcessInstanceDetailsTestWihtoutParentProcessInstance() {
+        when(processRuntimeDataServiceMock.getProcessInstance(any())).thenReturn(null);
+        presenter.setProcessInstance(processInstanceSummary);
+
+        verify(view, times(1)).setParentProcessInstanceIdText("", false);
```
Comment (cristianonicolai) [inline jbpm-wb-process-runtime/jbpm-wb-process-runtime-client/src/test/java/org/jbpm/workbench/pr/client/editors/instance/details/ProcessInstanceDetailsTabPresenterTest.java:123]: times(1) is the default anyway, you dont need to set.
```suggestion
        verify(view).setParentProcessInstanceIdText("", false);
```

**Comment by bxf12315** [comment]
depend on https://github.com/kiegroup/droolsjbpm-integration/pull/2193

File: jbpm-wb-process-runtime/jbpm-wb-process-runtime-client/src/test/java/org/jbpm/workbench/pr/client/editors/instance/details/ProcessInstanceDetailsTabPresenterTest.java (hunk 1 above)
Comment (bxf12315) [reply jbpm-wb-process-runtime/jbpm-wb-process-runtime-client/src/test/java/org/jbpm/workbench/pr/client/editors/instance/details/ProcessInstanceDetailsTabPresenterTest.java]: Done

**Review by nmirasch — APPROVED** [review]

**Comment by prashuts** [comment]
jenkins do fdb

**Comment by bxf12315** [comment]
jenkins do fdb

**Comment by prashuts** [comment]
@bxf12315  Error while starting process instance for the generated FDB - please find the detail in ticket https://issues.redhat.com/browse/JBPM-9347

**Comment by bxf12315** [comment]
jenkins do fdb

**Comment by bxf12315** [comment]
jenkins do fdb

**Comment by bxf12315** [comment]
jenkins do fdb

**Review by prashuts — APPROVED** [review]
Looks fine

**Pull request merged**